src/lib/openapi/validate.ts

```diff
@@ -38,3 +39,16 @@ export const validateSchema = <S = SchemaId>(
         };
     }
 };
+
+export const throwOnInvalidSchema = <S = SchemaId>(
+    data: object,
+    schema: S,
+): void => {
+    const validationErrors = validateSchema(schema, data);
+    if (validationErrors) {
+        throw fromOpenApiValidationErrors(
+            data,
+            validationErrors.errors as [ErrorObject, ...ErrorObject[]],
```

Comment:
More idiomatic pattern is
```
const [firstError, ...otherErrors] = validationErrors.
 throw fromOpenApiValidationErrors(
            data,
            [firstError, ...otherErrors]);
```

Reply:
Thanks, addressed in https://github.com/Unleash/unleash/pull/7616/commits/a056b0a9485b28656731595eaf23e0663c5ef43b